bootstrap/node.go

```diff
@@ -31,22 +31,27 @@ func NewNode(
 	nodePublicKey primitives.Ed25519PublicKey,
 	nodePrivateKey primitives.Ed25519PrivateKey,
 	federationNodes map[string]config.FederationNode,
-	blockSyncCommitTimeoutMillisec uint32,
+	blockSyncCommitTimeoutMillis uint32,
 	constantConsensusLeader primitives.Ed25519PublicKey,
 	activeConsensusAlgo consensus.ConsensusAlgoType,
-	benchmarkConsensusRoundRetryIntervalMillisec uint32, // TODO: move all of the config from the ctor, it's a smell
+	benchmarkConsensusRoundRetryIntervalMillis uint32, // TODO: move all of the config from the ctor, it's a smell
 	transport gossipAdapter.Transport,
+	belowMinimalBlockDelayMillis uint32,
```

Comment:
`NewNode` is now officially taking too many arguments. We need to decide, ASAP, how to handle passing config to `Node` from `main` and tests. @talkol any ideas?

Reply:
Correct, but it will not be resolved as part of this PR. Opened issue #74.

Reply:
Please take the time to handle #74 as well. Boy scout rule applies - leave the code better than how you found it. :)